### ISSUE TITLE
Pro source access option

### ISSUE THREAD

**Issue by tj** [apex/up#504]
Just an idea, but I imagine larger companies would feel more comfortable if they had access to the source/repo, just in case they need to tweak something or expand upon it. I'm not sure what the pricing would look like, need to think some more.

**Comment by kaihendry** [comment]
I'm interested to just read & learn from your source code ;)

**Comment by pctj101** [comment]
Yeah usually just small tweaks in a hurry when something simple needs to be changed.  Might be an undesirable few hours of hacking, but better than being stuck without a solution when AWS does something crazy or "an exception" needs to be made.
